### PR TITLE
python3Packages.jiwer: init at 3.0.1

### PR DESCRIPTION
**Pull request by GaetanLepage**
###### Description of changes

Add python package [jiwer](https://github.com/jitsi/jiwer)

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/jiwer/default.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  version = "3.0.1";
+
+  src = fetchFromGitHub {
+    owner = "jitsi";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-bH5TE6mcSG+WqvjW8Sd/o5bCBJmv9zurFEG2cVY/vYQ=";
+  };
+
+  format = "pyproject";
```
[the remark this answers is not in the document]
Comment (GaetanLepage) [reply pkgs/development/python-modules/jiwer/default.nix]: Like this ?

[the remark this answers is not in the document]
Comment (wegank) [reply pkgs/development/python-modules/jiwer/default.nix]: Yes.

**Comment by GaetanLepage** [comment]
There seem to be a bug when trying `nixpkgs-review rev HEAD`.
It is not due to this PR though as it also fails on master.

**Comment by wegank** [comment]
> There seems to be a bug when trying `nixpkgs-review rev HEAD`.
> It is not due to this PR though as it also fails on master.

cc #218870 (`pkgs.mysql-client` is an alias).

**Comment by GaetanLepage** [comment]
> cc #218870 (`pkgs.mysql-client` is an alias).

Oh Ok... Can we merge then ? We cannot do anything here right ?

File: pkgs/development/python-modules/jiwer/default.nix [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+  propagatedBuildInputs = [
+    rapidfuzz
+    click
+  ];
+
+  pythonImportsCheck = [ "jiwer" ];
+
+  meta = with lib; {
+    description = "JiWER is a simple and fast python package to evaluate an automatic speech recognition system";
```
Comment (dotlambda) [inline pkgs/development/python-modules/jiwer/default.nix:33]: > `meta.description` should not start with the package name.

(https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes)

Comment (wegank) [reply pkgs/development/python-modules/jiwer/default.nix:33]: #224953

**Pull request merged**